.github/ISSUE_TEMPLATE/new_feature_request.yml

```diff
@@ -0,0 +1,20 @@
+name: New Feature
+description: File some details about a new feature
+title: '[New Feature]: '
+labels: ['enhancement', 'help wanted']
+body:
+  - type: textarea
+    id: request-message
+    attributes:
+      label: What are you requesting?
```

Comment:
Requesting is also a bit non beginner friendly, should be "What affect will this have on the existing code base" or something